File: rest/sip-in/get-acl-list/get-acl-list.5.x.rb
require 'rubygems' # not necessary with ruby 1.9 but included for completeness
require 'twilio-ruby'

# put your own credentials here
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

# set up a client to talk to the Twilio REST API
@client = Twilio::REST::Client.new(account_sid, auth_token)

@acl_list_sid = 'AL32a3c49700934481addd5ce1659f04d2'

@ip_acl_list = @client.sip
                      .ip_access_control_lists(@acl_list_sid)
                      .fetch

puts @ip_acl_list.friendly_name

File: rest/sip-in/get-acl-list/get-acl-list.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioRestClient(account_sid, auth_token)

ip_access_control_list = client.sip \
    .ip_access_control_lists \
    .get("AL32a3c49700934481addd5ce1659f04d2")
print(ip_access_control_list.friendly_name)
